client/coral-admin/src/actions/community.js

```diff
@@ -0,0 +1,44 @@
+import qs from 'qs';
+
+import {
+  FETCH_COMMENTERS_REQUEST,
+  FETCH_COMMENTERS_SUCCESS,
+  FETCH_COMMENTERS_FAILURE,
+  SORT_UPDATE,
+  COMMENTERS_NEW_PAGE
+} from '../constants/community';
+
+import {base, getInit, handleResp} from '../helpers/response';
+
+export const fetchCommenters = (query = {}) => (dispatch) => {
+  dispatch(requestFetchCommenters());
+  fetch(`${base}/user?${qs.stringify(query)}`, getInit('GET'))
+    .then(handleResp)
+    .then(({result, page, count, limit, totalPages}) => {
+      dispatch({
+        type: FETCH_COMMENTERS_SUCCESS,
+        commenters: result,
+        page,
+        count,
+        limit,
+        totalPages
+      });
+    })
+    .catch((error) => {
```

Comment:
also (and same with the `then`) we can remove the `{ }`. Returning a value there will have no effect in this case